Source/Diagnostics/WarpXOpenPMD.H

```diff
@@ -88,7 +88,7 @@ public:
 
   ~WarpXOpenPMDPlot();
 
-  void SetStep(int ts);
+  bool SetStep(int ts);
```

Comment:
Can you please add a doxygen string here that describes the behaviour and meaning of the return value?